docs/data/toolpad/building-ui/canvas-and-inspector.md

```diff
@@ -0,0 +1,43 @@
+# Canvas
+
+<p class="description">
+    Canvas and Inspector are the two main areas when it comes to UI building
+</p>
+
+## Working with canvas
+
+Building UI is as easy as dragging component from the library and dropping it on the canvas.
+
+Once you have a few components on the canvas you can easily tweak them by doing following actions:
```

Comment:
```suggestion
Once you have a few components on the canvas you can tweak them by doing following actions:
```

I would avoid the overuse of adjectives like "easily". It doesn't add any information and it [may come off as condescending](https://jameshfisher.com/2017/02/22/dont-use-simply/).